src/SFA.DAS.PSRService.Web/Views/Shared/_LayoutNoMenu.cshtml

```diff
@@ -75,71 +75,28 @@
         <p>GOV.UK uses cookies to make the site simpler. <a href="https://www.gov.uk/help/cookies">Find out more about cookies</a></p>
     </div>
 
-    <header role="banner" id="global-header" class="with-proposition">
-        <div class="header-wrapper">
-            <div class="header-global">
-                <div class="header-logo">
-                    <a href="https://www.gov.uk/" title="Go to the GOV.UK homepage" id="logo" class="content">
-                        <img src="/assets/images/gov.uk_logotype_crown_invert_trans.png" width="36" height="32" alt=""> GOV.UK
-                    </a>
-                </div>
-            </div>
-            <div class="header-proposition">
-                <div class="content">
-                    <nav id="proposition-menu">
-                        <a id="proposition-name" href="@ViewData["RootUrl"]">Manage apprenticeships</a>
-                    </nav>
-                </div>
-            </div>
-        </div>
-    </header>
+    @Html.Partial("_Header", Html.GetHeaderViewModel(true))
```

Comment:
Might be good to pass hideNav: true as a names argument to the GetHeaderViewModel()

Reply:
Ok